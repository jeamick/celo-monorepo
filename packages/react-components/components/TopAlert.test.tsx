import TopAlert from '@celo/react-components/components/TopAlert'
import * as React from 'react'
import { Text } from 'react-native'
import * as renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <TopAlert height={36} backgroundColor={'red'} visible={true}>
      <Text>Hi</Text>
    </TopAlert>
  )
  expect(tree).toMatchSnapshot()
})
