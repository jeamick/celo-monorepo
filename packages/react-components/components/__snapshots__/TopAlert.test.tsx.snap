// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "red",
      "height": 36,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "center",
        "opacity": 1,
        "paddingHorizontal": 25,
      }
    }
  >
    <Text>
      Hi
    </Text>
  </View>
</View>
`;
